src/openseadragon.js

```diff
@@ -1346,22 +1346,20 @@ function OpenSeadragon( options ){
             compositeOperation:                null, // to be passed into each TiledImage
 
             // DRAWER SETTINGS
-            drawer:                            ['webgl', 'canvas', 'html'], // prefer using webgl, context2d, fallback to html
+            drawer:                            ['webgl', 'canvas', 'html'], // prefer using webgl, then canvas (i.e. context2d), then fallback to html
                 /**
                  * drawerOptions dictionary.
                  * @type {Object} drawerOptions
-                 * @property {Object} webgl - options if the WebGLDrawer is used.
-                 * Set 'continuousTileFresh: true' if tile data is modified programmatically
-                 * by filtering plugins or similar.
-                 * @property {Object} context2d - options if the CanvasDrawer is used
-                 * @property {Object} html - options if the HTMLDrawer is used
-                 * @property {Object} custom - options if a custom drawer is used
+                 * @property {Object} webgl - options if the WebGLDrawer is used. No options are currently supported.
+                 * @property {Object} canvas - options if the CanvasDrawer is used. No options are currently supported.
+                 * @property {Object} html - options if the HTMLDrawer is used. No options are currently supported.
+                 * @property {Object} custom - options if a custom drawer is used. No options are currently supported.
                  */
             drawerOptions: {
                 webgl: {
-                    continuousTileRefresh: false,
+
                 },
-                context2d: {
+                canvas: {
```

Comment:
I am actually not sure about this change, since asi we discussed with cache PR, _canvas_ is ambiguous as WebGL API is also operated through a canvas instance...

Reply:
I still think that `canvas` is the more recognizable term. 

That said, I'm aware I also said we should use `context2d` as our actual data type, since it's easier to get a canvas from a context than it is to get a context from a canvas: https://github.com/openseadragon/openseadragon/pull/2407#issuecomment-1821370263

I don't know if those two statements conflict with each other.

Reply:
I don't think they necessarily conflict. I think `context2d` is a fine data type, and I agree that `Context2dDrawer` is more awkward than `CanvasDrawer`